cfme/tests/cloud/test_security_groups.py

```diff
@@ -0,0 +1,59 @@
+import fauxfactory
+import pytest
+
+from cfme.cloud.provider.openstack import OpenStackProvider
+from cfme.utils.wait import TimedOutError
+
+pytestmark = [
+    pytest.mark.usefixtures('setup_provider'),
+    pytest.mark.provider([OpenStackProvider], scope="module")
+]
+
+
+@pytest.yield_fixture(scope='module')
+def sec_group(appliance, provider):
+    security_group = appliance.collections.security_group
+    try:
+        sec_groups = security_group.create(name=fauxfactory.gen_alphanumeric(),
+                                           description=fauxfactory.gen_alphanumeric(),
+                                           provider=provider,
+                                           wait=True
+                                           )
+    except TimedOutError:
+        pytest.fail('Timed out creating Security Groups')
+    yield sec_groups
+    sec_groups.delete(wait=True)
+
+
+@pytest.mark.tier(3)
+def test_security_groups_crud(sec_group):
+    """ This will test whether it will create new Security Group and then deletes it.
+    Steps:
+        * Select Network Manager.
+        * Provide Security groups name.
+        * Provide Security groups Description.
+        * Select Cloud Tenant.
+        * Also delete it.
+    """
+    assert sec_group.exists
+    sec_group.delete(wait=True)
+    assert not sec_group.exists
```

Comment:
**REQUIRED**
I don't see any U from CRUD here. Please add TODO in `SecurityGroup` and in this test.